lib/awesome_spawn.rb

```diff
@@ -67,9 +89,8 @@ def logger
   # @raise [NoSuchFileError] if the `command` is not found
   # @return [CommandResult] the output stream, error stream, and exit status
   # @see http://ruby-doc.org/core/Kernel.html#method-i-spawn Kernel.spawn
-  def run(command, options = {})
-    bad_keys = (options.keys.flatten & [:in, :out, :err]).map { |k| ":#{k}" }
-    raise ArgumentError, "options cannot contain #{bad_keys.join(", ")}" if bad_keys.any?
+  def run(*opts)
```

Comment:
would we want a different method? `run_multiple` or `run_piped`?

Reply:
Unless you are thinking of calling out to either `run_multiple`/`run_piped` from the method when necessary, I would say no to this suggestion.  This is a public method, and one of the goals of this PR was to not muck with the existing way the public interface worked (hence a lot of what I was testing in my specs).